src/wbi/remote/templates/pyhello.jinja

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+#SBATCH --job-name=wbi-hello
```

Comment:
Converted both jinja templates to dynamically name jobs using `wbi-{{templatename}}`. Also changed the `template` parameter to `template_name` because `template` was being overwritten on `line 33` of `remote/remote.py`.